projects/2019_11_13_pkt_insight.md

```diff
@@ -0,0 +1,147 @@
+* **F_NS_BALANCE** Network steward address balance
+* **F_NS_TX** List of transactions by network steward address - with links to relevant project pages
+
+#### Mobile support
+* The explorer will provide *basic functionality* on an android mobile device browser including at least:
+  * **F_M_BALANCE** Ability to check address balance
+  * **F_M_EARNING** PKT earned in the past day
+  * **F_M_LASTBLK** Ability to check when the last block was produced
+* Best effort will be made to support other mobile devices to the same level
+
+## Success criteria
+* **S_CI** Codebase passing automated continuous integration build
+* **S_INSTALL** At least one successful independent installation of the explorer
+* **S_FEATURES** At least 90% of the described features achieved
+
+## Payments
+All payments shall be made to `p7Gdf7YhaxSkWm6u6yU452S6C9mJpuTfwu`
+
+## Milestones
+The following are milestones for the progress of the project by which the network steward
+can evaluate the success of the project.
+
+### M0 Kickoff
+At the kickoff of the project, the network steward will grant 10mn PKT to the applicant and
+the project will begin.
+
+### M1
+Milestone 1 will focus on improvements to the address page, front page and block detail page.
+Features expected to be completed include:
+
+* **F_ADDR_EARNING**
+* **F_ADDR_EARNING_CHART**
+* **F_ADDR_MINING_TAB**
+* **F_FRNT_BW**
+* **F_FRNT_CSS**
+* **F_FRNT_CHART**
+* **F_BLK_PCVER**
+* **F_BLK_CRYPTOCYCLES**
+* **F_BLK_BANDWIDTH**
+
+Furthermore, Milestone 1 must be validated by a continuous integration build using a publicly
+accessible continuous integration service such as travis-ci (**S_CI**).
+
+Upon the completion of Milestone 1, a report will be written and the applicant will seek
+approval from the network steward. No payment will occur at this time but if Milestone 1 is
+accepted but Milestone 2 does not complete successfully, the network steward will pay the
+applicant 5mn PKT on top of the initial 10mn PKT and the steward will retain the final 10mn
+PKT of the budget.
+
+### M2
+Milestone 2 will focus on improvements to the blocks page, the creation of the network steward
+page, and improved mobile support. Specific features which will be used to validate Milestone 2
+will include:
+
+* **F_NS_CURRENT**
+* **F_NS_TALLY**
+* **F_NS_RUNNERSUP**
+* **F_NS_BURNED**
+* **F_NS_BALANCE**
+* **F_NS_TX**
+* **F_BLKS_RELAGE**
+* **F_BLKS_RETARGET**
+* **F_BLKS_NXT_RETARGET**
+* **F_M_BALANCE**
+* **F_M_EARNING**
+* **F_M_LASTBLK**
+
+Upon completion of Milestone 2, a report will be written and the applicant will seek
+approval from the network steward. Upon approval, payment of 15mn PKT will be issued, covering
+Milestone 1 and Milestone 2.
+
+At this point the project will reach the end of it's explicit funding, however the applicant
+commits to the long term maintanence of the PKT insight software and the network steward commits
+to continuing interest in the evolution of the software.
```

Comment:
As above, the completion of the second milestone should result in a payment of 10m PKT, on the basis that the 10m PKT initial + 5m PKT first milestone payment will already have been settled.